.k8/yamls-v1/ssv-node-v2-1-deployment.yml

```diff
@@ -119,16 +119,12 @@ spec:
           value: "false"
         - name: DISCOVERY_TYPE_KEY
           value: "discv5"
-        - name: NETWORK
```

Comment:
I don't understand why in some we add network = jato and in some we delete network entirely ?

Reply:
We used to have `NETWORK` (beacon network name, e.g. `prater`) and `NETWORK_ID` (domain, e.g. `0x00003011`). But `NETWORK_ID` was set only on stage nodes, it was not set on prod nodes, which used a default value. 

Now we don't need to configure both as they are stored in the network descriptor. So we remove them. But instead, we need to set a network name. We set it only on stage nodes because the prod nodes use the default value (`mainnet`), this logic is similar to what we had with `NETWORK_ID`. For setting a network name we reuse the `NETWORK` field.